src/charts/line.js

```diff
@@ -188,7 +188,7 @@ define(function(require){
             dataByTopic,
             dataByDate,
 
-            dateLabel = 'date',
+            dateLabel = 'fullDate',
```

Comment:
I would actually keep here the label as 'date' and change the fixture data. I had to do the same on BCReact